src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/DiagnosticListener.cs

```diff
@@ -42,6 +42,8 @@ public static IObservable<DiagnosticListener> AllListeners
         {
             get
             {
+                GC.KeepAlive(HttpHandlerDiagnosticListener.s_instance);
```

Comment:
I am not claiming we support NET45, since I am going back to verify 4.5. I am just putting NET46 and NETFX on it.